### DISCUSSION TITLE
The model is gaining water (WaterBalanceError is positive)

### DISCUSSION THREAD

**Discussion by aniketgupta2009** [NCAR/hrldas#75] [Q&A] [answered]
Hi,

I am getting a strange error in Noah-MP Version 5 which is "The model is gaining water (WaterBalanceError is positive)". Create forcing is working perfectly using the NLDAS2 Grib files and model writes output for the first 519 timesteps (hourly). It shows the above-mentioned error at the 520th timestep. I tried using the single core as well as multi-cores but the problem persist. A screenshot of the error is attached to this issue.

Please let me know how I could resolve the problem.

Thanking You.

DATA: NLDAS2 Grib Model forcing File
Year: 2017
Timesteps: Hourly

![Noah-MP5_ERROR](https://github.com/NCAR/hrldas/assets/50705068/1e0bf58b-f936-4095-8f92-d6c6eb4a2e60)

**Comment by cenlinhe** [comment]
Thank you for your question. What are your Noah-MP namelist options?

**Comment by aniketgupta2009** [comment]
Hi,

Please find the attached name list options.
[namelist.hrldas.txt](https://github.com/NCAR/hrldas/files/11669598/namelist.hrldas.txt)

**Comment by cenlinhe** [comment, marked as answer]
The water balance issue is caused by too large model timestep. Please change NOAH_TIMESTEP in the namelist from 3600 to 1800 or 900. Typically, 1800 (i.e., half hour) should be enough for most applications.

**Comment by aniketgupta2009** [comment]
Hi Cenlin,

Thank you so much for resolving the issue. The model is working fine now with NOAH_TIMESTEP at 1800.

**Reply by cenlinhe** [reply]
Great!
